src/Operation/Pipe.php

```diff
@@ -26,31 +24,31 @@ final class Pipe extends AbstractOperation
     /**
      * @pure
      *
-     * @return Closure(...callable(Iterator<TKey, T>):Generator<TKey, T, mixed, mixed>):Closure(Iterator<TKey, T>):Iterator<TKey, T>
+     * @return Closure(callable(iterable<TKey, T>): iterable<TKey, T> ...): Closure(iterable<TKey, T>): iterable<TKey, T>
```

Comment:
I expanded the type here from `Iterator` to `iterable` to allow for the `All` usage, which returns an `array`

Reply:
Nice one!